### DISCUSSION TITLE
NginxProxyManager as docker redirecting to containers on same host

### DISCUSSION THREAD

**Discussion by EddyVanEm** [NginxProxyManager/nginx-proxy-manager#3236] [Q&A]
Hi everyone,
I am having issues to redirect requests to containers on the same host. If I redirect to any host outside of the localhost it will work perfectly. Redirecting to any container of the same host just doesn't work. I tried the local ip adress of the host. I tried localhost I tried the container ip. Nothing seems to work. Does anyone have an idea what I am doing wrong? 

Any help appreciated!

**Comment by lastsamurai26** [comment]
can you Ping inside of your NginxProxyManager Container the Other Container you want to rewrite?

If not you need to connect all container which need to be pingable or asseable from nginx into one Network, maybe create a proxynetwork for that
